shanoir-uploader/src/main/java/org/shanoir/uploader/gui/MainWindow.java

```diff
@@ -128,6 +126,7 @@ public class MainWindow extends JFrame {
 	public String studyDate = "";
 	public String modality;
 	JScrollPane scrollPaneUpload;
+	public JButton btndeleteFinishedUploads;

```

Comment:
Hi @LaurentPV, might be nice to respect Java btnDeleteFinishedUploads way of variable naming, also very minor, but it might happen, that in the future Checkstyle will refuse this. Thank you and with kind regards, Michael